percy/snapshot.py

```diff
@@ -83,26 +83,29 @@ def percy_snapshot(driver, name, **kwargs):
         print(f'{LABEL} {e}')
 
 # Take screenshot on driver
-def percy_automate_screenshot(driver, name, **kwargs):
+def percy_automate_screenshot(driver, name, options = None, **kwargs):
     if not is_percy_enabled(): return
+    if options is None:
+        options = {}
```

Comment:
Can't we do `options = { }` in the function's default arg

Reply:
Linter breaks and other reason is https://stackoverflow.com/a/26320917/8238359

Reply:
Interesting, didn't know this. Nice